### PR TITLE
Remove mask-hoarding section

### PR DESCRIPTION
**Pull request by jaysonvirissimo**
This section was too vague, used questionable reasoning, and seemed unjustified.

> Do not hoard masks

The word "hoard" is doing a lot of work here, but is vague. What constitutes hoarding? What about buying one respirator for each member of the family? Is that hoarding? What if you buy 60 to wear on the way to work on the train, and you expect to use 1 per day and have to wear them for 2 months?

> Healthcare workers desperately need them to care for you and your loved ones.

This is certainly true, but whether you having your own mask means healthcare workers will have too few depends on the context. If your region has no shortage of masks, then you having your own does not impede healthcare workers from safely doing their job. At the very least, this should be highly qualified, since although the CDC is not recommending that Americans wear them, many East Asian governments are making them _mandatory_. For example, Dr. Pak-Leung Ho [partially credits](https://twitter.com/lwcalex/status/1235091542219448321) universal mask-wearing for Hong Kongs relative success in containing the outbreak. Also, see [this article](https://www.reuters.com/article/us-china-health-masks-safety/to-mask-or-not-to-mask-confusion-spreads-over-coronavirus-protection-idUSKBN1ZU0PH) which claims:

> In China, where nearly 10,000 cases have been reported so far, Zunyou Wu, chief epidemiologist at the Chinese Center for Disease Control and Prevention, told state broadcaster that people need to wear masks when taking public transport.

> Masks are NOT the most useful thing you can do to protect yourself

The mere fact that something is not the most useful thing you can do is not a good argument for not doing that thing. The same can be said of everything (except one) in the "Do" section of this very website. We do not tell parents to not use car seats merely because not driving at all is safer still.

>  they’re hard to wear properly,

Whether they are "hard" to wear properly is a very subjective claim. Hard for whom? Construction workers and painters don't seem to have much trouble with them and they aren't typically highly trained medical professionals. There are detailed instructions on most manufacturer websites as well as videos on their proper use on YouTube ([here](https://www.youtube.com/watch?v=zoxpvDVo_NI) is an example). According to the BBC, many [people are washing their hands wrong](https://www.bbc.com/reel/video/p084zgcf/why-you-re-washing-your-hands-wrong) too, yet we (properly, IMO) continue to recommend that people wash their hands.

> Also for some people, wearing a mask makes it more likely that they will touch their face.

For some people this is no doubt true, but is it true for most people? In fact, there are experts that say exactly the opposite of this. [This](https://www.nytimes.com/2020/03/05/health/stop-touching-your-face-coronavirus.html) NYT article quotes Dr. Justin Ko as saying that "they can be quite helpful for providing a physical barrier against touching the nose or mouth."

### CODE REVIEW

File: _content/en/03-dont/05-do-not-hoard-masks.md [hunk 1]
```diff
@@ -1,4 +0,0 @@
-{:.dont}
-## Do not hoard masks
```
Comment (jaysonvirissimo) [inline _content/en/03-dont/05-do-not-hoard-masks.md]: The word "hoard" is doing a lot of work here, but is vague. What constitutes hoarding? What about buying one respirator for each member of the family? Is that hoarding? What if you buy 60 to wear on the way to work on the train, and you expect to use 1 per day and have to wear them for 2 months? Would that qualify?

**Comment by jaysonvirissimo** [comment]
This is related to [this](https://github.com/flattenthecurve/guide/issues/34) issue, but falls short of recommending mask-wearing (which the CDC does not, but many East Asian government do).

**Comment by matiasgarciaisaia** [comment]
I'm not an authority here at all, but I think China & other eastern countries tend to have a more common use of masks - in which case the supply chain is already used to the demand, and governments recommending using them are not proposing something _that_ revolutionary.

In western countries is mainly the opposite. If we all go rushing for masks, there simply won't be enough - the difference between usual demand and everyone trying to get one would be way higher than in eastern countries. So, to avoid shortages for the doctors (I know it's happening here in Buenos Aires), the recommendation to not _hoard_ masks seems reasonable.

The meaning of hoard may intentionally be vague. A doctor trying to buy 60 masks would be standard use; me buying 60 masks would be hoarding.

Last thing - I guess if you were one of the people that _needs to_ get masks (doctors, cashier at a store that won't close, etc), you would probably know that this general piece of advise doesn't apply to you since you're an exception.

But, again - those are my two cents. I've labeled `sci-review` so anyone that actually know about this can comment :)

Thanks for opening the debate, @jaysonvirissimo!

**Comment by anarchodoc** [comment]
hi, masks shouldn't be used by the general public in regular everyday usage - they don't add anything and are a waste of resources.

**Comment by anarchodoc** [comment]
NB, I could probably write an article about the utility (or not!) of masks at some point. There was a good BMJ article a few years ago about _if_ they actually work (no evidence to show they do) . ... mostly they protect other people from you (rather than you from other people)

**Comment by anarchodoc** [comment]
Facemasks for the prevention of infection in healthcare and community settings
https://www.bmj.com/content/350/bmj.h694 

Basically, more research needed - probably the thing that has an impact is hand washing and not facemasks.

**Comment by jaysonvirissimo** [comment]
**To be clear, this proposed change is to remove a section I feel is too vague and contains fallacious reasoning, not necessarily to replace it with a pro-face mask section.**

**If we decide that removing it isn't the right call, I still think we should update it to make it less vague and use better arguments than it currently contains (assuming those exist).**

But, to respond to @anarchodoc:
> hi, masks shouldn't be used by the general public in regular everyday usage - they don't add anything and are a waste of resources.

This statement is apparently contradicted by other authorities. Dr. Pak-Leung Ho, head of Centre for Infection at HKU Medicine, "highlighted universal mask-wearing as one of the reasons widespread outbreak didn't occur". See [this](https://twitter.com/lwcalex/status/1235091542219448321) tweet for more context. Angela Rasmussen, virologist at the Center for Infection & Immunity at Columbia University, said that "Situations that require a mask are when you are in a crowd...or if you are caring for a sick person". See [this](https://www.reuters.com/article/us-china-health-masks-safety/to-mask-or-not-to-mask-confusion-spreads-over-coronavirus-protection-idUSKBN1ZU0PH) article for the surrounding context.

**Comment by jaysonvirissimo** [comment]
> hi, masks shouldn't be used by the general public in regular everyday usage - they don't add anything and are a waste of resources.

**Comment by anarchodoc** [comment]
hi jay,

> **To be clear, this proposed change is to remove a section I feel is too vague and contains fallacious reasoning, not necessarily to replace it with a pro-face mask section.**
> 
> **If we decide that removing it isn't the right call, I still think we should update it to make it less vague and use better arguments than it currently contains (assuming those exist).**
> 
I agree with removing it.

> But, to respond to @anarchodoc:
> 
> > hi, masks shouldn't be used by the general public in regular everyday usage - they don't add anything and are a waste of resources.
> 
> This statement is apparently contradicted by other authorities.
> 
> Dr. Pak-Leung Ho, head of Centre for Infection at HKU Medicine, "highlighted universal mask-wearing as one of the reasons widespread outbreak didn't occur". See [this](https://twitter.com/lwcalex/status/1235091542219448321) tweet for more context.
> 

Disagree completely. Expert opinion is WAY DOWN in terms of medical evidence. The BMJ article I quoted if you read it properly says there is no evidence at all that masks do anything.

What _is_ perhaps likely is that the behaviour change associated with wearing a mask helps reduce infection risk - but it is _not_ the wearing the mask itself that reduces infection risk.

> Angela Rasmussen, virologist at the Center for Infection & Immunity at Columbia University, said that "Situations that require a mask are when you are in a crowd...or if you are caring for a sick person". See [this](https://www.reuters.com/article/us-china-health-masks-safety/to-mask-or-not-to-mask-confusion-spreads-over-coronavirus-protection-idUSKBN1ZU0PH) article for the surrounding context.
> 

Again - opinion: there is no evidence to suggest that masks work.  


> > NB, I could probably write an article about the utility (or not!) of masks at some point. There was a good BMJ article a few years ago about if they actually work (no evidence to show they do) . ... mostly they protect other people from you (rather than you from other people)
> 
> Saying that they work _because they protect other people from you_ is just another way of saying they work _for the purposes of flattening the curve_. This is especially true if, like the LA Times suggests, [silent spreaders are fueling the coronavirus pandemic](https://www.latimes.com/science/story/2020-03-17/how-silent-spreaders-are-fueling-the-coronavirus-pandemic).

Again, sorry but BS. They may work in specific contexts (e.g. surgery) but they do not help to flatten the curve when worn in general everyday/outdoors environment as you seem to be suggesting.

**Comment by anarchodoc** [comment]
NB, just to provide some qualification - I'm a medical doctor and also a trained epidemiologist.

**Comment by anarchodoc** [comment]
> NB, just to provide some qualification - I'm a medical doctor and also a trained epidemiologist.

And also in the scientific review group of FTC (I'm in slack)

**Comment by jaysonvirissimo** [comment]
Additional qualification: I have **absolutely no expertise** in medicine or epidemiology. I'm just a computer  programmer that noticed some poor logic in one of the sections of this wonderful and desperately needed website.

**Comment by tpudlik** [comment]
I too know nothing about medicine or epidemiology, but I did read the sources linked to above.  The BMJ article does not claim that there is "no evidence to show" facemasks work, or that "probably the thing that has an impact is hand washing and not facemasks."  It states *exactly the opposite* in the abstract (emphasis mine):

> Several randomised clinical trials of facemasks have been conducted in community and healthcare settings, using widely varying interventions, including mixed interventions (such as masks and handwashing), and diverse outcomes. Of the nine trials of facemasks identified in community settings, in all but one, facemasks were used for respiratory protection of well people. **They found that facemasks and facemasks plus hand hygiene may prevent infection in community settings, subject to early use and compliance.**

**Comment by paul-hammant** [comment]
@mverzilli you closed two of my pull requests and asked me to chime in here. 

1. Can we AT LEAST qualify "Don't hoard masks" as "don't hoard surgical masks".
2. Delete the masks don't lie verbiage
3. Add a Make-a-mask section / buy-a-cloth-mask if you're a civililain

**Comment by anarchodoc** [comment]
> I too know nothing about medicine or epidemiology, but I did read the sources linked to above. The BMJ article does not claim that there is "no evidence to show" facemasks work, or that "probably the thing that has an impact is hand washing and not facemasks." It states _exactly the opposite_ in the abstract (emphasis mine):
> 
> > Several randomised clinical trials of facemasks have been conducted in community and healthcare settings, using widely varying interventions, including mixed interventions (such as masks and handwashing), and diverse outcomes. Of the nine trials of facemasks identified in community settings, in all but one, facemasks were used for respiratory protection of well people. **They found that facemasks and facemasks plus hand hygiene may prevent infection in community settings, subject to early use and compliance.**

I suggest reading the full article (try via sci-hub).

There's another article here which addresses N95 masks versus surgical masks:
https://onlinelibrary.wiley.com/doi/full/10.1111/jebm.12381

(open access) 

My interpretation is that it basically suggests neither are any good at preventing inhalation of virus but probably wearing something over your mouth/nose helps you to stop transmitting something that is on your fingertips/hands.

**Comment by jaysonvirissimo** [comment]
@mverzilli: looks an unrelated merge accidentally undeleted this section. See [here](https://github.com/flattenthecurve/guide/blob/master/_sections/act_and_prepare/en/03-dont/05-do-not-hoard-masks.md). Lemme know if you want another PR or if you can just delete it yourself.

**Comment by mverzilli** [comment]
Thanks @jaysonvirissimo! I had seen a heads up from @paul-hammant that the section was still up and was a bit surprised by that. Thank you for putting the time to investigate.

@paul-hammant sorry for closing the other PRs, I wanted to keep the conversation around masks in a single place because it gets a bit difficult to get to a coherent result otherwise. 

Here's what I'll do in around 8 hours once I'm out of my day job and can get back to this project:

1. I'll make sure this section effectively goes away.
1. I'll open a PR trying to recover and synthesize the contents of the 2 PRs I closed before. I'll also include a summary of what's been discussed here. 
1. I'll mention you both so you can review and suggest improvements if necessary.
1. I'll also try to get the reviewers involved, since we'll need approvals for the next content.

@jaysonvirissimo and @paul-hammant any help advancing with the 2 first steps is super welcome, otherwise I'll take care of it when I get back to working on this project at the end of the day.

Apologies for the inconvenience, there's still many moving parts and we're trying to get things organized as fast as we can. Thank you for your contributions and patience 🙏 !

**Comment by jaysonvirissimo** [comment]
@mverzilli: merghing [this](https://github.com/flattenthecurve/guide/pull/220) PR should allow you to knock out (1) pretty quickly. I'll be around tonight (MST) for help with (3). 👍

**Comment by mverzilli** [comment]
Merged! Thank you so much!

**Comment by paul-hammant** [comment]
How did the merge get smushed previously - someone did a force-push ?

**Pull request merged**